DeepSpace/Messages/RepairShipRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace DeepSpace.Messages
+{
+    public class RepairShipRequest
+    {
+        [JsonProperty(PropertyName = "commandCode")]
+        public string CommandCode { get; set; }
+
+        [JsonProperty(PropertyName = "health")]
+        public double Health { get; set; }
```

Comment:
As this is a UIless game players/developers will call this method directly. By giving them a health variable you're effectively asking them how much they'd like to heal. I'd much rather this heals by... let's say 10 points per call for the time being. That means a player needs to sit and repair for a little while before they can fight again